Modules/PermModule.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Threading.Tasks;
+using Discord.Commands;
+using snipetrain_bot.Models;
+using snipetrain_bot.Services;
+using Discord;
+using MongoDB.Driver;
+
+namespace snipetrain_bot.Modules
+{
+    [Group("user")]
+    public class PermModule : ModuleBase
+    {
+        private readonly IPermService _Permservice;
+        public PermModule(IPermService permService)
+        {
+            _Permservice = permService;
+        }
+        [Command("kick")]
+        [RequireUserPermission(GuildPermission.KickMembers)]
+        [RequireBotPermission(GuildPermission.KickMembers)]
+        public async Task KickAsync(IGuildUser user, [Remainder] string reason)
+        {
+            try
+            {
+                var adminName = Context.User.ToString();
+                var date = DateTime.Now;
+                var kickedUser = user.ToString();
+                var userid = user.Id;
+
+                var DBinfo = new PermissionSchema
+                {
+                    AdminName = adminName,
+                    Date = date,
+                    Reason = reason,
+                    User = kickedUser,
+                    UserId = userid
+                };
+                await user.KickAsync(reason);
+                await _Permservice.AddKickAsync(DBinfo);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.ToString());
```

Comment:
This is a command, you are not replying in the event of an exception, this might confuse the user as to what is happening. (this applies to all commands in this file.)